runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBoxOverlay.java

```diff
@@ -76,13 +84,17 @@
 		this.config = config;
 		this.skill = skill;
 		this.icon = icon;
+		this.xpPauseState = xpPauseState;
 		panel.setBorder(new Rectangle(BORDER_SIZE, BORDER_SIZE, BORDER_SIZE, BORDER_SIZE));
 		panel.setGap(new Point(0, XP_AND_PROGRESS_BAR_GAP));
 		panel.setPreferredSize(new Dimension(PANEL_PREFERRED_WIDTH, 0));
 		iconXpSplitPanel.setBorder(XP_AND_ICON_COMPONENT_BORDER);
 		iconXpSplitPanel.setBackgroundColor(null);
 		iconXpSplitPanel.setPreferredSize(new Dimension(PANEL_PREFERRED_WIDTH, 0));
 		getMenuEntries().add(new OverlayMenuEntry(RUNELITE_OVERLAY_CONFIG, OPTION_CONFIGURE, "XP Tracker overlay"));
+		getMenuEntries().add(new OverlayMenuEntry(RUNELITE_OVERLAY, OPTION_REMOVE, skill.getName()));
+		getMenuEntries().add(new OverlayMenuEntry(RUNELITE_OVERLAY, OPTION_RESET, skill.getName()));
+		getMenuEntries().add(new OverlayMenuEntry(RUNELITE_OVERLAY, OPTION_TOGGLE_TRACKING, skill.getName()));
```

Comment:
you can extend OverlayMenuEntry to return Pause/Resume based on XpPauseState here